docs/ocis/migration.md

```diff
@@ -0,0 +1,160 @@
+---
+title: "Migration"
+date: 2021-03-16T16:17:00+01:00
+weight: 41
+geekdocRepo: https://github.com/owncloud/ocis
+geekdocEditPath: edit/master/docs/ocis
+geekdocFilePath: migration.md
+---
+
+
+## Migration
+
+The migration happens in subsequent stages while the service is online.
+
+### User Stories
+As an admin I need to avoid downtime.
+As an admin I want to migrate certain groups of users before others.
+As a user I need a seamless migration and not loose data by any chance.
+
+### Stage-0
+Is the pre-migration stage having a functional ownCloud 10 instance
+
+### Stage-1
+Introduce OpenID Connect to server and Clients
+
+### Stage-2
+Introduce new web ui, let end users get used to it voluntarily
+
+### Stage-3
+start oCIS backend and make read only tests on existing data using owncloud storage driver and database backed share manager
```

Comment:
Where is the "existing data" supposed to come from? There was no data migrated yet, right?

Reply:
True. Are you referring to some dummy data, like we have with a fresh new installation of oC10? Should adding this data to a fresh new oCIS installation be a task then?

Reply:
I think I clarified this.